### DISCUSSION TITLE
Trying to raise awareness of openHasp with Node-Red

### DISCUSSION THREAD

**Discussion by smcgann99** [HASwitchPlate/openHASP#546] [General]
I have been aware of this project for some time, but until the arrival of the newer 4" switch plates didn't see any hardware that I liked.
This project is a great fit with NR but doesn't get the attention it deservers in that community. 
So I open a new thread - https://discourse.nodered.org/t/using-openhasp-with-node-red/81457

There is a lot of scope for collaboration, with NR flows and possibly custom "nodes" to aid integration between the two. 

Anyway thanks for this great project, and I will be posting my progress on NR as I go along ;-)

**Comment by smcgann99** [comment]
I added a small demo flow for node red if anyone is interested, it's actually quite simple to integrate openHasp.

https://discourse.nodered.org/t/using-openhasp-with-node-red/81457

**Comment by thatkide** [comment]
Nice, I use Node-Red to update my plates.  It originally started with a bunch of HASPone plates then I updated my nodes to support Openhasp.  Most of my nodes are functions, one for listening to HA and sending commands to openhasp and the other to listen to openhasp and send commands to HA.  The goal was for me to have a scalable way to handle many plates installed throughout the house.  It'll be nice to contribute and see what others have done.
